disperser/cmd/dataapi/main.go

```diff
@@ -155,7 +156,7 @@ func RunDataApi(ctx *cli.Context) error {
 	return runServer(server, logger)
 }
 
-func runServer[T dataapi.ServerInterface](server T, logger logging.Logger) error {
+func runServer[T serverv2.ServerInterface](server T, logger logging.Logger) error {
```

Comment:
ServerInterface is a shared one, to make this "runServer" template function work for both v1 and v2